src/less/uw-ui-toolkit/my-uw/buttons.less

```diff
@@ -20,7 +20,7 @@
   padding:2px 5px;
 }
 .btn-default {
-  background-color:transparent;
+  //background-color:transparent;
```

Comment:
Oops, sorry. I really am not paying attention. I did not see that this was in the MyUW subdir. I typically would not touch this at all.